fasta/pFasta/tools.go

```diff
@@ -72,9 +98,11 @@ func faToPfa(input fasta.Fasta, start int, end int) PFasta {
 		} else if base == dna.T {
 			answer.Seq[idx] = pDna.Float32Base{A: 0, C: 0, G: 0, T: 1}
 		} else if base == dna.N {
-			answer.Seq[idx] = pDna.Float32Base{A: 0, C: 0, G: 0, T: 0}
+			//answer.Seq[idx] = pDna.Float32Base{A: 0, C: 0, G: 0, T: 0} // Raven's change: I think N should be all 0.25, not all 0
```

Comment:
Alert Sarah and Riley of this change
Use the dna/pDna/pDna.go > DnaToPdna(base dna.Base) function instead?